addons/mail/tests/test_res_partner.py

```diff
@@ -185,3 +185,99 @@ def test_res_partner_merge_wizards(self):
         all_msg = p2_msg_ids_init + p1_msg_ids_init + p1_msg1
         self.assertEqual(len(p2.message_ids), len(all_msg) + 1, 'Should have original messages + a log')
         self.assertTrue(all(msg in p2.message_ids for msg in all_msg))
+
+    def test_res_partner_get_starred_count_with_empty_body(self):
+        new_user = self.env['res.users'].create({
+            'email': 'francoisemb@test.be',
+            'login': 'benjam',
+            'name': 'Benjam Prejent',
+        })
+        new_partner = new_user.partner_id
+        new_msg = new_partner.message_post(body="TestingMessage", message_type="comment")
```

Comment:
It is very strange to use partner as both "message" and "business model". I would maybe move the test in test_mail and use a test model as source of messages / notifications / starred, and check partner starred count update.

Reply:
Hello @tde-banana-odoo ,
Just updated the PR according to your reviews! (+ moved everything in `test_mail` and now using `mail.test.ticket` test model)
May I have your opinion on these changes please? :)
Thanks